src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -156,7 +199,13 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
-    private bool SyncToPr(uint pr, StringBuilder executionResult)
+    private string GetRepoPath(uint repoId)
```

Comment:
Why would you create a private repo in this scenario? The implication of doing this is that someone will be making direct edits to the PRService server to do a private run. 

IMO, one is much better off creating a dev branch and asking the PR service to key off id={somenumber}&branch=origin/dev/tom. It is much more friction-free than having one of our devs log into the box and creating a repo.